src/components/select/select.template.tsx

```diff
@@ -42,11 +43,13 @@ export const SelectTemplate: FunctionalComponent<SelectTemplateProps> = props =>
         'limel-select--required': props.required,
         'limel-select--invalid': !isValid,
         'limel-select--empty': !hasValue,
+        'limel-select--with-helper-text': typeof props.helperText === 'string',
     };
 
     return (
         <div class={classList}>
             <SelectValue {...props} hasValue={hasValue} />
+            <HelperText text={props.helperText} />
```

Comment:
This is a little annoying:

![image](https://user-images.githubusercontent.com/154725/72965414-87d12c80-3dbc-11ea-9aaf-ff23903c1d67.png)

But that's down to MDC of course. I wonder if we could clearfix the helper-text or something… or maybe we could clearfix the bottom inside of the form frame… 🤔

Reply:
I can experiment a bit with that later.

Reply:
Hmm yeah maybe it should be a bit closer to the line of the select as well? I'm trying to compare it to the input field which seems to be a bit closer. Also, I'm not sure we want to have the line around the form, but that's related to the field template. However, I guess this will still be an issue if there are other fields below this one, they will probably flow into each other

Reply:
Pushed some changes to the positioning. The issue seems to be that the height of the field is hard coded by material and doesn't take the helper text into account

Reply:
Ah, yeah, there are definitely some differences. The text is further away here, and not indented like for input-field.

I saw you just pushed some changes for this, so I'll take a look at it with those.

Reply:
I just remembered though that the field is not styled by material since we are not using their select component haha :upside_down_face: Anyway, it looks a bit better now... :smile:

Reply:
Haha, yeah, I had also forgotten that, but I was about to comment that it doesn't seem like it's styled by material, at least not completely 😄

It looks much better now! And this should be a step on the way towards adding a text for when a select is required but has been left empty, as well 👍

Reply:
Well isn't that beautiful! 🥳

![image](https://user-images.githubusercontent.com/154725/72967846-8efb3900-3dc2-11ea-9895-208190b459dc.png)